#!/usr/bin/env python
PACKAGE = "me5413_world"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_throttle", double_t, 1, "Maximum throttle value. Default: 0.5", 0.5, 0.1, 10.0)
gen.add("throttle_gain", double_t, 1, "Throttle gain. Default: 0.1", 0.1, 0.1, 1.0)
gen.add("robot_length", double_t, 1, "Length of the robot (for pure pursuit algorithm). Default: 0.5", 0.5, 0.1, 10.0)
gen.add("lookahead_distance", double_t, 1, "Default lookahead distance. Default: 0.5", 0.5, 0.1, 10.0)

exit(gen.generate(PACKAGE, "path_tracker_node", "path_tracker"))
